### PR TITLE
fix(proxy): Updated mps server and mps router ports as env variables

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**

### CODE REVIEW

File: internal/proxy/proxy.go [hunk 1]
```diff
@@ -89,8 +89,14 @@ func (s Server) forward(conn net.Conn, destChannel chan net.Conn) {
...
 				//call to database to get the mps instance
-				destination = mpsdb.Query(guid)
+				instance := mpsdb.Query(guid)
+				if (instance != "") {
+					parts := strings.Split(destination, ":")
+					parts[0] = instance
+					destination = parts[0] + ":" + parts[1]
+				}
 			}
+			// log.Println("destination :", destination)
```
Comment (rsdmike) [inline internal/proxy/proxy.go:99]: remove

File: cmd/main.go [hunk 2]
```diff
@@ -8,12 +8,25 @@ import (
...
 
 func main() {
+	
 	flag.Parse()
 
-	p := proxy.NewServer(":8003", "mps:3000")
-	log.Println("Proxying from " + p.Addr + " to " + p.Target)
+	routerPort := os.Getenv("MPS_ROUTER_PORT")
+	if routerPort == "" {
+		log.Println("MPS_ROUTER_PORT env is not set")
```
Comment (rsdmike) [inline cmd/main.go:20]: lets change this to just PORT

File: cmd/main.go [hunk 3]
```diff
@@ -8,12 +8,25 @@ import (
...
+		routerPort = "8003"
+	}
+	mpsPort := os.Getenv("MPS_SERVER_PORT")
```
Comment (rsdmike) [inline cmd/main.go:23]: lets just call this MPS_PORT

File: cmd/main.go [hunk 4]
```diff
@@ -8,12 +8,25 @@ import (
 	"flag"
 	"log"
 	"mps-lookup/internal/proxy"
+	"os"
 )
 
 func main() {
+	
 	flag.Parse()
```
Comment (rsdmike) [inline cmd/main.go:16]: remove since we aren't parsing command line flags

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: internal/proxy/proxy.go (hunk 1 above)
Comment (madhavilosetty-intel) [reply internal/proxy/proxy.go]: done

File: cmd/main.go (hunk 2 above)
Comment (madhavilosetty-intel) [reply cmd/main.go]: done

Comment (madhavilosetty-intel) [reply cmd/main.go]: done

File: cmd/main.go (hunk 3 above)
Comment (madhavilosetty-intel) [reply cmd/main.go]: done

File: cmd/main.go (hunk 4 above)
Comment (madhavilosetty-intel) [reply cmd/main.go]: done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**